notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<ul><li>Maybe a concrete example for point 1 would be useful. (e.g.: "If set x to y this means that v percent of the request will be logged" or something like this explaining what the logging frequency means)</li><li>I am not sure I understand point 2</li><li>Maybe some more explanation for point 4 and point 5 are needed</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>